providers/resend/src/lib/resend.provider.ts

```diff
@@ -17,6 +17,7 @@ export class ResendEmailProvider implements IEmailProvider {
     private config: {
       apiKey: string;
       from: string;
+      senderName: string;
```

Comment:
Sender name is an optional property of CredentialsDTO. In the implementation provided that is not taken in account and it is assumed it will always be present. And that's not right.